pages/dkp/kommander/2.1/install/configuration/index.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: layout.pug
+navigationTitle: Configuration
+title: Configuration
+menuWeight: 30
+excerpt: How to configure a Kommander installation
+beta: false
+enterprise: false
+---
+
+You can configure Kommander during the initial installation, and also post-installation using the Kommander CLI.
+
+## Initializing a configuration file
+
+To begin configuring Kommander, run the following command to initialize a default configuration file:
+
+```bash
+kommander install --init > kommander.yaml
+```
+
+## Configuring applications
+
+After you have a default configuration file, you can then configure each `app` either inline **or** by referencing another `yaml` file. The configuration values for each `app` correspond to the Helm Chart values for the application.
```

Comment:
Do we have somewhere that documents where to find the helm charts?  If so, I would suggest we add a link here.  If not, please consider adding this to this PR.    Thank you!!!!

Reply:
I didn't find any existing docs, adding a small note in 7f877ee5769731ee35f68e4362c8389ce4f79712. We don't really have a great way to make this discoverable today, the way i've done it is by referencing the HelmRelease for each application.